smite-lib/SerializationModels.fs

```diff
@@ -24,11 +24,15 @@ type NamespaceDefinition =
     { Namespace : string []
       Models : ModelDefinition [] }
 
-type FilespaceDefinition =
+type SingleNamespaceFilespaceDefinition =
     { Namespace : string []
       Filespace : string []
       Models : ModelDefinition [] }
 
+type MultiNamespaceFilespaceDefinition =
```

Comment:
Technically single SingleNamespaceFilespaceDefinition is special case of MultiNamespaceFilespaceDefinition. SingleNamespaceFilespaceDefinition will be removed and MultiNamespaceFilespaceDefinition  will be renamed to FilespaceDefinition.